ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -125,9 +146,7 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
         }
     }
 
-#if os(iOS)
     @objc public weak var delegate: TableViewHeaderFooterViewDelegate?
```

Comment:
is it possible to also deprecate the delegate so it's removed when UITextItemInteraction is